app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  before_action :require_login, only: [:index, :destroy]
+
+  def index
+    @users = User.all.order(created_at: :desc)
+    @user = current_user
+  end
+
+  def destroy
+    if current_user.admin?
+      @user_nonadmin = User.find(params[:id])
+      @user_nonadmin.destroy
```

Comment:
Does this need to be an instance variable if you're not passing it to the view? Also, you need some flow control to notify the user in case something failed on delete.